docs/commands_draft.txt

```diff
@@ -7,64 +7,83 @@ We will call a file containing such commands a "hdb script".
 
 Commands in the system's terminal
 ------------------------------------
-	hashesdb:
-		Starts an interactive dialog
-		
-	hashesdb help
+hashesdb
+	Starts an interactive dialog (REPL)
 
-	hashesdb help command
+hashesdb -h|--help
+	Prints help
 
-	hashesdb version
+hashesdb -v|--version
+	Prints tool's version
 
-	hashesdb about
+hashesdb about
+	Prints information about the project
 
-	hashesdb create database_path [--overwrite]
+hashesdb schema
+	Prints schema documentation
 
-	hashesdb -u database_path:
-		Starts an interactive dialog.
-		Sets the .db file located at the given path as the database we are currently using.
+hashesdb create database_path [--overwrite]
+	Creates a hashesdb database at a given path

```

Comment:
don't have a separate syntax for it.
Since every command takes a `-d db_file` option, this one should also use the same.

Reply:
Okey, this probably applies to #79 too. I will make the appropriate changes.

Reply:
I made the appropriate changes in PR #92.
I will also change this documentation file.